Library/Homebrew/extend/ENV/super.rb

```diff
@@ -167,7 +167,7 @@ def determine_isystem_paths
   end
 
   def determine_include_paths
-    PATH.new(keg_only_deps.map(&:opt_include)).existing
+    PATH.new(deps.map(&:opt_include)).existing
```

Comment:
the same environment variable as in https://github.com/Homebrew/brew/pull/2970#discussion_r130244910?